libs/@guardian/react-crossword/src/components/Controls.tsx

```diff
@@ -149,6 +167,8 @@ const AnagramHelper = (props: ButtonProps) => {
 			onClick={toggleAnagramHelper}
 			data-link-name="Show anagram helper"
 			{...props}
+			role="none"
+			aria-hidden="true"
```

Comment:
There's no need for both as `aria-hidden` will remove the element from the accessibility tree so it's role won't be exposed anyway: https://www.scottohara.me/blog/2018/05/05/hidden-vs-none.html